ups/validation_data_jointcal.table

```diff
@@ -0,0 +1,2 @@
+setupRequired(obs_lsstSim)
+setupRequired(obs_cfht)
```

Comment:
Are both really "required"?  Can't I run jointcal on the CFHT data without obs_lsstSim?  Then you mean `setupOptional`.